cmd/influxd/inspect/type_conflicts/check_schema.go

```diff
@@ -0,0 +1,145 @@
+package typecheck
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	"github.com/influxdata/influxdb/v2/kit/cli"
+	"github.com/influxdata/influxdb/v2/tsdb"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zapcore"
+)
+
+type TypeConflictChecker struct {
+	Path          string
+	SchemaFile    string
+	ConflictsFile string
+	Logger        *zap.Logger
+
+	logLevel zapcore.Level
+}
+
+func NewCheckSchemaCommand(v *viper.Viper) (*cobra.Command, error) {
```

Comment:
We should probably check for the existence of any `tsdb.FieldsChangeFile` and warn or abort.  This change should then be backported to master-1.x (and 1.11 if that branch has been created).  My error in not updating this code after the incremental update to fields.idx (https://github.com/influxdata/influxdb/pull/23728 and https://github.com/influxdata/influxdb/pull/23701).

Reply:
Added a new commit, I tested it by just touching a `fields.idxl` file next to a `fields.idx` file before running the command.
<img width="1520" alt="Screenshot 2022-11-18 at 13 07 02" src="https://user-images.githubusercontent.com/26460069/202774621-a1f3ed46-3883-4b98-bd86-2ad5b17bfa94.png">